lib/factory_bot_rails/reloader.rb

```diff
@@ -4,21 +4,22 @@
 
 module FactoryBotRails
   class Reloader
-    def initialize(app, config)
+    def initialize(app)
```

Comment:
Is there anything we should include in the docs now there is not a config to be passed in?

Reply:
This class is not meant for public use, so changing the API should be fine.